### PR TITLE
[CSL-2246] Add support for new Items API `onMissing` parameter

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: src/modules/catalog.js [hunk 1]
```diff
@@ -579,6 +589,7 @@ class Catalog {
    * @param {object} parameters - Additional parameters for variation details
    * @param {object[]} parameters.variations - A list of variations with the same attributes as defined in the Variation schema resource (https://docs.constructor.io/rest_api/items/variations/#variation-schema)
    * @param {boolean} [parameters.force=false] - Process the request even if it will invalidate a large number of existing variations
+   * @param {string} [parameters.onMissing] - Defines the strategy for handling items which are present in the file and missing in the system. IGNORE silently prevents adding them to the system, CREATE creates them, FAIL fails the ingestion in case of their presence. Defaults to FAIL
```
Comment (stanlp1) [inline src/modules/catalog.js:592]: Same as previous.

**Review by stanlp1 — APPROVED** [review]
Minor comment about comments. But otherwise lgtm.

File: src/modules/catalog.js [hunk 2]
```diff
@@ -266,6 +266,7 @@ class Catalog {
    * @param {object} parameters - Additional parameters for item details
    * @param {object[]} parameters.items - A list of items with the same attributes as defined in the Item schema resource (https://docs.constructor.io/rest_api/items/items/#item-schema)
    * @param {boolean} [parameters.force=false] - Process the request even if it will invalidate a large number of existing items
+   * @param {string} [parameters.onMissing] - Defines the strategy for handling items which are present in the file and missing in the system. IGNORE silently prevents adding them to the system, CREATE creates them, FAIL fails the ingestion in case of their presence. Defaults to FAIL
```
[the remark this answers is not in the document]
Comment (esezen) [reply src/modules/catalog.js]: Good catch. Fixed here: [470a022](https://github.com/Constructor-io/constructorio-node/pull/143/commits/470a02245475bf5da365b7265a15f8840a726cc4)

**Review by stanlp1 — APPROVED** [review]
Looks good

**Pull request merged**